src/QsCompiler/CommandLineTool/Options.cs

```diff
@@ -50,6 +50,14 @@ public class CompilationOptions : Options
         HelpText = "Additional properties to populate the AssemblyConstants dictionary with. Each item is expected to be of the form \"key:value\".")]
         public IEnumerable<string> AdditionalAssemblyProperties { get; set; }
 
+        [Option("runtime", Required = false, SetName = CODE_MODE, 
+        HelpText = "Specifies the classical capabilites of the runtime. Determines what QIR profile to compile to.")]
+        public AssemblyConstants.RuntimeCapabilities RuntimeCapabilites { get; set; }
+
+        [Option("build-exe", Required = false, Default = false, SetName = CODE_MODE,
+        HelpText = "Specifies whether to build a Q# command line application.")]
+        public bool MakeExecutable { get; set; }
```

Comment:
I assume you mean non-standalone, but with a custom driver. No, actually this would be true even if you have a custom driver. The goal eventually would be to not have C# and Q# code in the same project, but instead have a separate project when you want to have a custom driver. Things will remain backwards compatible for now, but a warning should be generated if there is a exe project with no Q# entry point (e.g. because there is a C# driver in the same project).

Reply:
I updated the diagnostic to be a bit more meaningful.